docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -449,7 +449,9 @@ spec:
 > use a custom MAC address for your VM instances. If you think you may
 > be affected by any of issues mentioned above, consider changing the
 > default interface type to `masquerade`, and disabling the `bridge`
-> type for pod network, as shown in the example above.
+> type for pod network, as shown in the example above. Setting a static 
+> mac address in your yaml file for multiple vms should not cause problems 
+> as the MAC address is used for VM to pod networking.
```

Comment:
Thanks for your PR!

I have several observations/change requests:

* I think this snippet should move to a new Note: block under the "masquerade" section that starts around line 484.
* "setting a static mac address" may not convey what you intend. (My understanding could be wrong) Would "Reusing the same static MAC address for multiple VMs" work for you here?
* MAC should be all caps to match the rest of the document.